### PR TITLE
Fix `clippy::use-self`` warning in ` src/main.rs`

### PR DESCRIPTION
**Pull request by pmnoxx**
`ClippyCmd` warnings gets generated due to addition of `clippy::use-self`. This PR fixes that.

```
warning: unnecessary structure name repetition
  --> src/main.rs:99:9
   |
99 |         ClippyCmd {
   |         ^^^^^^^^^ help: use the applicable keyword: `Self`
   |
   = note: `-W clippy::use-self` implied by `-W clippy::nursery`
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#use_self
```

---

changelog: none

### CODE REVIEW

**Review by xFrednet — APPROVED** [review]
Thank you for the Pull Request, the change looks good to me. Before merging this, I'm interested how you found this. Our Dogfood test should have caught this in the CI, have you executed some other command? :upside_down_face:

**Comment by xFrednet** [comment]
That's a good hint, looking at the dogfood test it seems like it also ignores the sources under test. That's good to know! Thank you :upside_down_face: 

@bors r+

**Comment by xFrednet** [comment]
BTW, I just noticed that your GH profile isn't linked to the commit. The email is probably different, this is not a problem, and it can be merged as it is, just wanted to point that out in case that was an accident. I'll wait for a response until I retry :upside_down_face: 

(I fixed the changelog entry)

**Comment by matthiaskrgr** [comment]
I think `-W clippy::nursery` is not part of the clippy-self-check of the ci setup.

**Comment by pmnoxx** [comment]
> BTW, I just noticed that your GH profile isn't linked to the commit. The email is probably different, this is not a problem, and it can be merged as it is, just wanted to point that out in case that was an accident. I'll wait for a response until I retry upside_down_face
> 
> (I fixed the changelog entry)

Thanks for noticing. How did you find out that it's not linked properly?

**Comment by pmnoxx** [comment]
I changed setting in my github profile, the email address should be linked properly now.

**Comment by xFrednet** [comment]
You can look at the commit in this PR if your picture shows up, and you can click on it, then it's correctly linked. (Like it is now)

@bors retry

**Pull request merged**
